### ISSUE TITLE
Update semantic type for custom_axes parameter

### ISSUE THREAD

**Issue by ElDeveloper** [qiime2/q2-emperor#78]
In order to keep things consistent, it would be great to update the semantic type for the [`custom_axes`](https://github.com/qiime2/q2-emperor/blob/master/q2_emperor/plugin_setup.py#L50) parameter to be a Metadata Column.

**Comment by thermokarst** [comment]
Yes! I think we might need to think a bit more about the `List` side of things here
